src/main/java/net/dancier/kikeriki/DumpingMessageHandler.java

```diff
@@ -0,0 +1,18 @@
+package net.dancier.kikeriki;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Component
+public class DumpingMessageHandler implements MessageHandler {
+  @Override
+  public void handleFoo(String key, MessageFoo foo) {
+    log.info("handling foo -> key: %s, message: %s", key, foo);
+  }
+
+  @Override
+  public void handleBar(String key, MessageBar bar) {
+    log.info("handling bar -> key: %s, message: %s", key, bar);
+  }
+}
```

Comment:
Without at least this dummy handler the service just did not start, and so we have it running in the test-env.